sql/src/test/java/io/crate/executor/transport/task/elasticsearch/ESGetTaskTest.java

```diff
@@ -90,4 +91,24 @@ public void testContextCreation() throws Exception {
         assertThat(subContext, notNullValue());
         assertThat(subContext, instanceOf(ESGetContext.class));
     }
+
+    @Test
+    public void testContextKill() throws Exception {
+        UUID jobId = UUID.randomUUID();
+        ESGetTask task = new ESGetTask(
+                jobId,
+                mock(Functions.class),
+                mock(ProjectionToProjectorVisitor.class),
+                mock(TransportMultiGetAction.class),
+                mock(TransportGetAction.class),
+                node,
+                jobContextService);
+
+        JobExecutionContext jobExecutionContext = jobContextService.getContext(jobId);
+        ExecutionSubContext subContext = jobExecutionContext.getSubContext(node.executionNodeId());
+        subContext.kill();
+
+        assertThat(task.result().size(), is(1));
+        assertThat(task.result().get(0).isCancelled(), is(true));
```

Comment:
assert that  after kill `jobExecutionContext.getSubContextOrNull(id)` returns null